amd_openvx_extensions/amd_rpp/include/vx_ext_rpp.h

```diff
@@ -321,7 +321,7 @@ SHARED_PUBLIC vx_node VX_API_CALL vxExtrppNode_ContrastbatchPD(vx_graph graph,vx
  * \returns A node reference <tt>\ref vx_node</tt>. Any possible errors preventing a
  * successful creation should be checked using <tt>\ref vxGetStatus</tt>.
  */
-SHARED_PUBLIC vx_node VX_API_CALL vxExtrppNode_Copy(vx_graph graph, vx_image pSrc, vx_image pDst);
+SHARED_PUBLIC vx_node VX_API_CALL vxExtrppNode_CopybatchPD(vx_graph graph, vx_image pSrc, vx_image pDst);
```

Comment:
@fiona-gladwin I see develop branch build breaks. Is this the fix to that?
Please verify to make future PRs do not break anything

Reply:
Yes Lakshmi This PR fixes the build issues wrt PR [#1141 ](https://github.com/GPUOpen-ProfessionalCompute-Libraries/MIVisionX/pull/1141)